adoc/deployment-vmware.adoc

```diff
@@ -275,19 +275,32 @@ terraform plan
 terraform apply
 ----
 
-==== Setup by Hand
+==== Setup with {ay}
```

Comment:
This heading should be one level up (it is not related to templates) : `=== Setup with {ay}`